### PR TITLE
Solved little error in configuration code

### PR DESCRIPTION
**Pull request by sammychinedu2ky**
I had to make the conditional statement come immediately after initializing the _array variable with the config values.
I also removed the early setting of array values in the ArrayExample class for proper indexing of config values

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=25389) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=25389) before we can accept your contribution.<br/><br/>:x: sammychinedu2ky [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=25389)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetCore.Docs?pullRequest=25389) it.</sub>

File: aspnetcore/fundamentals/configuration/index/samples/6.x/ConfigSample/Pages/Array.cshtml.cs [hunk 1]
```diff
@@ -18,12 +18,11 @@ public ArrayModel(IConfiguration config)
 
         public ContentResult OnGet()
         {
+            _array = _c.GetSection("array").Get<ArrayExample>();
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/configuration/index/samples/6.x/ConfigSample/Pages/Array.cshtml.cs:21]: _array = Config.GetSection("array").Get<ArrayExample>();

**Comment by sammychinedu2ky** [comment]
Thanks @Rick-Anderson I've made the change

**Comment by sammychinedu2ky** [comment]
Any update @serpent5 😅

**Review by serpent5 — APPROVED** [review]

**Comment by serpent5** [comment]
Thanks for your contribution, @sammychinedu2ky. Be sure to check [What's New](https://docs.microsoft.com/en-us/aspnet/core/whats-new) early next month, where you'll see yourself listed as a valued contributor.

**Comment by sammychinedu2ky** [comment]
Thanks a lot 😊.

**Pull request merged**